src/ASM/LR/ASMu2D.C

```diff
@@ -2606,7 +2606,7 @@ void ASMu2D::generateThreadGroups (const Integrand& integrand, bool silence,
     LR::generateThreadGroups(projThreadGroups, projBasis.get());
   if (silence || threadGroups[0].size() < 2) return;
 
-  std::cout <<"\nMultiple threads are utilized during element assembly.";
+  IFEM::cout <<"\nMultiple threads are utilized during element assembly.";
```

Comment:
Although it is inside `#ifdef SP_DEBUG` shouldn't L2612 and L2614 also use IFEM::cout then? (In case someone one day tries to activate these print out in parallel runs).

Reply:
indeed, fixed.